core/templates/dev/head/pages/splash.html

```diff
@@ -8,30 +8,114 @@
   Oppia
 {% endblock subtitle %}
 
+{% block header_js %}
+  {{ super() }}
+
+  <style>
+    html, body {
+      background-color: #AFD2EB;
+    }
+  </style>
+{% endblock %}
+
 {% block navbar_breadcrumb %}
+  <ul class="nav navbar-nav oppia-navbar-breadcrumb">
+    <li>
+      <!--
+      <span class="oppia-navbar-breadcrumb-separator"></span>
+      Splash
+      -->
+    </li>
+  </ul>
 {% endblock navbar_breadcrumb %}
 
 {% block content %}
-  <div class="oppia-gallery-container">
-    <div class="oppia-gallery-banner-container" style="background-color: #888;">
-      <div class="oppia-gallery-banner-content" style="height: 100%; width: 100%;">
-        Hi, I&rsquo;m Oppia.
-        <br>
-        <span style="font-size: 0.8em;">
-          This splash page needs to be replaced.
-        </span>
-      </div>
+
+  <div id="oppia-splash-section-one" class="text-center">
+    <h1 class="oppia-splash-h1">Think outside the books.</h1>
+    <div class="oppia-splash-background">
+      <img src="images/splash/Economics.svg" class="oppia-splash-back-images" />
```

Comment:
Replace "/>" with ">" -- img tags are void elements, and it's a codebase convention to omit the "/". http://stackoverflow.com/a/7854998

Also, all these images don't seem to be visible...